doc/Release_Build_Creation.rst

```diff
@@ -1,16 +1,32 @@
-=======================
-Release Build Creation
-=======================
+===========================
+Compile & Build Cortx Stack
+===========================
 
 This file consists of the procedure that should be followed to generate the release build outside the Seagate network using `cortx-build <https://github.com/orgs/Seagate/packages/container/package/cortx-build>`_ docker image. 
 
-***************
-Procedure
-***************
+*****************************************
+Specifications to setup Virtual Machine
+*****************************************
 
-#. Setup a `CentOS 7.8.2003 <http://isoredirect.centos.org/centos/7.8.2003/isos/x86_64/>`_ system.
+- Single-Node VM deployment
 
-   - You can use a Virtual Machine (VM) also.
+  - Please create VM(s) with at least 1 cores (vCPU) and 4GB of RAM.
+  - Ensure the VM is created with minimum of total 9 attached raw disks (additional disks apart from main bootable disk) per node
```

Comment:
Sorry, this needs to be returned to what it was before.  Does CORTX run correctly with 1 disk?  If so, this should be removed.  There is nothing in the CORTX software (motr, S3, nowhere) that requires 9 disks.

Reply:
done